cos-fleetshard-sync/src/main/java/org/bf2/cos/fleetshard/sync/resources/ProcessorStatusUpdater.java

```diff
@@ -0,0 +1,57 @@
+package org.bf2.cos.fleetshard.sync.resources;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.bf2.cos.fleet.manager.model.ProcessorDeploymentStatus;
+import org.bf2.cos.fleetshard.api.ManagedProcessor;
+import org.bf2.cos.fleetshard.sync.FleetShardSyncConfig;
+import org.bf2.cos.fleetshard.sync.client.FleetManagerClient;
+import org.bf2.cos.fleetshard.sync.client.FleetManagerClientException;
+import org.bf2.cos.fleetshard.sync.client.FleetShardClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@ApplicationScoped
+public class ProcessorStatusUpdater {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProcessorStatusUpdater.class);
+    @Inject
+    FleetManagerClient fleetManagerClient;
+    @Inject
+    FleetShardClient connectorClient;
+    @Inject
+    FleetShardSyncConfig config;
+
+    public void update(ManagedProcessor processor) {
+        LOGGER.debug("Update processor status (name: {}, phase: {})",
+            processor.getMetadata().getName(),
+            processor.getStatus().getPhase());
+
+        fleetManagerClient.getProcessorDeployments(Long.parseLong(processor.getSpec().getDeploymentId()),
```

Comment:
I'm not very sure I get this right, I guess it is only a placeholder at this stage ?